src/extra/stats.js

```diff
@@ -16,8 +16,7 @@ Bootstrap.registerPlugin("stats", {
   },
 
   uninstall: function (three) {
-    document.body.removeChild(this.stats.domElement);
-
+    this.stats.domElement.remove()
```

Comment:
The stats plugin is added as a child of Threestrap's element, which is not necessarily the body element.

Reply:
ah, that is why I was hitting this! makes sense.